### DISCUSSION TITLE
Support for Dynamic Filtering in Subqueries?

### DISCUSSION THREAD

**Discussion by zhugw** [apache/superset#25146] [Q&A / Help]
Is Superset capable of supporting a SQL query like this, where the value of 'upc' is dynamic and equals the selected value from a filter dropdown?
```
SELECT DISTINCT province , city, shop_name from symd_yizi where shop_name not in (select distinct shop_name from symd_yizi where upc = 'xxx') t
```
<img width="473" alt="image" src="https://github.com/apache/superset/assets/4953871/44bf7d93-d487-424e-8696-d1961aef17b8">

**Comment by NevzatTaha** [comment]
you can check https://superset.apache.org/docs/configuration/sql-templating/#jinja-templates
